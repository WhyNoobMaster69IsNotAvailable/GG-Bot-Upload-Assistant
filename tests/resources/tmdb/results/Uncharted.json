{
    "page": 1,
    "results": [
        {
            "adult": false,
            "backdrop_path": "/aEGiJJP91HsKVTEPy1HhmN0wRLm.jpg",
            "genre_ids": [
                28,
                12
            ],
            "id": 335787,
            "original_language": "en",
            "original_title": "Uncharted",
            "overview": "",
            "popularity": 1918.903,
            "poster_path": "/tlZpSxYuBRoVJBOpUrPdQe9FmFq.jpg",
            "release_date": "2022-02-10",
            "title": "Uncharted",
            "video": false,
            "vote_average": 7.1,
            "vote_count": 2453
        },
        {
            "adult": false,
            "backdrop_path": null,
            "genre_ids": [
                27
            ],
            "id": 34168,
            "original_language": "en",
            "original_title": "Uncharted",
            "overview": "\"Uncharted\" is a story of a documentary film crew who crash on an island, when they were in hopes of finding a rare animal, a jaguar, that is believed to inhabit the jungle. Unfortunate circumstances arise when her charter plane goes down and crashes on a remote island.",
            "popularity": 12.724,
            "poster_path": "/1hR8UOQ3NCmNM45XEj2WmjWfvul.jpg",
            "release_date": "2009-01-01",
            "title": "Uncharted",
            "video": false,
            "vote_average": 3.6,
            "vote_count": 5
        },
        {
            "adult": false,
            "backdrop_path": "/zydjvavUeFb4kAuQORt3NFo3MzB.jpg",
            "genre_ids": [
                28,
                12
            ],
            "id": 940497,
            "original_language": "en",
            "original_title": "Uncharted: Live Action Fan Film",
            "overview": "Treasure hunter Nathan Drake finds himself captured while searching for the lost treasure of the Flor de la Mar (Flower of the Sea).",
            "popularity": 15.352,
            "poster_path": "/zsvJowd6wkoHVk4k94oxk5SdNGQ.jpg",
            "release_date": "2018-07-16",
            "title": "Uncharted: Live Action Fan Film",
            "video": false,
            "vote_average": 6.2,
            "vote_count": 12
        },
        {
            "adult": false,
            "backdrop_path": "/coYapkwiwWwachN36ZO3tjrBsa8.jpg",
            "genre_ids": [
                28,
                12,
                16
            ],
            "id": 271916,
            "original_language": "en",
            "original_title": "Uncharted: Eye of Indra",
            "overview": "In the run-up to the release of 'Uncharted 2: Among Thieves', Naughty Dog and Sony Computer Entertainment published 'Uncharted: Eye of Indra'.\r This motion comic is a prequel to the events shown in 'Uncharted: Drake's Fortune',",
            "popularity": 4.497,
            "poster_path": "/kw03vncbtgjgx9OZqpSZzdtC4x1.jpg",
            "release_date": "2009-10-22",
            "title": "Uncharted: Eye of Indra",
            "video": false,
            "vote_average": 4.5,
            "vote_count": 2
        },
        {
            "adult": false,
            "backdrop_path": "/57VPEZNTDMj1zNpW6l8rT8Jzx2M.jpg",
            "genre_ids": [
                99
            ],
            "id": 942512,
            "original_language": "en",
            "original_title": "Gerry Anderson: A Life Uncharted",
            "overview": "Gerry Andersonâ€™s successful career spanned 60 years. Released a decade after his death, this documentary draws on exclusive access to over 30 hours of previously unpublished interviews, in order to share the untold stories that defined his life and body of work.",
            "popularity": 2.987,
            "poster_path": "/lLSTjXqUgM1e8gvut00o3aH34v.jpg",
            "release_date": "2022-04-15",
            "title": "Gerry Anderson: A Life Uncharted",
            "video": false,
            "vote_average": 0,
            "vote_count": 0
        },
        {
            "adult": false,
            "backdrop_path": null,
            "genre_ids": [
                99
            ],
            "id": 626968,
            "original_language": "nl",
            "original_title": "Uncharted Amazon",
            "overview": "",
            "popularity": 0.634,
            "poster_path": "/mPShVWhpHUQVXpzpL4sF2uUfedy.jpg",
            "release_date": "2016-03-01",
            "title": "Uncharted Amazon",
            "video": false,
            "vote_average": 0,
            "vote_count": 0
        },
        {
            "adult": false,
            "backdrop_path": "/dX0QCBt0XSsPT4963WoAj6gZny.jpg",
            "genre_ids": [
                10749
            ],
            "id": 725288,
            "original_language": "en",
            "original_title": "Uncharted Seas",
            "overview": "After her drunken husband Tom brings home three cabaret women, Lucretia can no longer bear the abuse and turns to Arctic explorer Frank, who has long loved her and promised to come back to her whenever she needs his help.",
            "popularity": 0.6,
            "poster_path": "/qKMY2rXRdaR0pLZxE1IuRR8IXyW.jpg",
            "release_date": "1921-04-25",
            "title": "Uncharted Seas",
            "video": false,
            "vote_average": 0,
            "vote_count": 0
        },
        {
            "adult": false,
            "backdrop_path": null,
            "genre_ids": [
                14,
                12
            ],
            "id": 962352,
            "original_language": "en",
            "original_title": "Land Uncharted",
            "popularity": 0.6,
            "poster_path": null,
            "release_date": "",
            "title": "Land Uncharted",
            "video": false,
            "vote_average": 0,
            "vote_count": 0
        },
        {
            "adult": false,
            "backdrop_path": "/jEdokCOwNzU8P9Jd34by8I2Wn5o.jpg",
            "genre_ids": [
                12,
                99
            ],
            "id": 443220,
            "original_language": "en",
            "original_title": "Uncharted Lines",
            "overview": "Every climbing reward comes at the cost of a potential risk, but when you are a climbing pioneer, driven by the unknown and unexpected, you are willing to risk whatever it takes for the chance to find the perfect first ascent line. This is a story of exploration and discovery, a journey of friendship and solitude, a quest to fuel an obsessive passion. Starring Paul Robinson, Jimmy Webb, Daniel Woods, Chris Sharma and Niky Ceria.",
            "popularity": 0.6,
            "poster_path": null,
            "release_date": "2017-01-16",
            "title": "Uncharted Lines",
            "video": false,
            "vote_average": 0,
            "vote_count": 0
        },
        {
            "adult": false,
            "backdrop_path": null,
            "genre_ids": [
                99
            ],
            "id": 273156,
            "original_language": "en",
            "original_title": "Uncharted Waters",
            "overview": "Uncharted Waters is a feature-length documentary about Australian surfing legend, Wayne Lynch. It traces his upbringing in the seaside town of Lorne on Victoria's South-Western coast and his turbulent experiences as a 'conscientious objector' on the run from conscription and the Vietnam war. Ultimately it is about an extremely gifted individual with an intense connection to the Southern Ocean, whose approach to surfing has been a spiritual journey, often putting him at odds with the surfing sub-culture and society in general.",
            "popularity": 0.6,
            "poster_path": null,
            "release_date": "2013-10-26",
            "title": "Uncharted Waters",
            "video": false,
            "vote_average": 4.3,
            "vote_count": 2
        },
        {
            "adult": false,
            "backdrop_path": null,
            "genre_ids": [
                99
            ],
            "id": 329894,
            "original_language": "en",
            "original_title": "Uncharted Waters",
            "overview": "It was a stunning announcement: Bill Koch, winner of the 1992 America's Cup, wanted to form an all female sailing team to try to win the 1995 Cup. The bold experiment ended in heartbreak for the Mighty Mary's crew, but the twists and turns of their story are worth remembering.",
            "popularity": 0.6,
            "poster_path": null,
            "release_date": "2014-07-31",
            "title": "Uncharted Waters",
            "video": false,
            "vote_average": 0,
            "vote_count": 0
        },
        {
            "adult": false,
            "backdrop_path": null,
            "genre_ids": [],
            "id": 305293,
            "original_language": "en",
            "original_title": "Braaap 14: Uncharted",
            "overview": "Braaap 14 takes a turn into uncharted territory. Explore the minds of today's ground-breaking extreme winter sports athletes as they reveal secrets of their lives and places they adventure. \"Uncharted\" brings you the deepest powder, the steepest mountains and some of the best terrain on this back country adventure. Experience the cutting edge as we introduce you to the next level: the Timbersled Mountain Horse snow bike conversion kit. \"Uncharted\" provides a perfect mix of the best riders, X-Games, freestyle, boondocking and cliff dropping action that is simply off the charts. Hear from some of the riders as they test boundaries, pushing forward and covering new ground.",
            "popularity": 0.6,
            "poster_path": "/mPhLsh7EfF0OxwJzpCYP8pZLkDH.jpg",
            "release_date": "2014-09-17",
            "title": "Braaap 14: Uncharted",
            "video": true,
            "vote_average": 0,
            "vote_count": 0
        },
        {
            "adult": false,
            "backdrop_path": null,
            "genre_ids": [
                99
            ],
            "id": 373362,
            "original_language": "en",
            "original_title": "Star Trek: Enterprise - Uncharted Territory",
            "overview": "This 90 minute documentary features all-new interviews with the show's cast and crew as well as behind the scenes archival content providing fans an inside look at the challenges faced during the second season of ENTERPRISE as the writing staff and creators struggled to find the right tone and creative course for the series, ultimately developing the controversial XINDI story-arc which kicks off with the season finale: THE EXPANSE.",
            "popularity": 1.169,
            "poster_path": null,
            "release_date": "2013-08-20",
            "title": "Star Trek: Enterprise - Uncharted Territory",
            "video": false,
            "vote_average": 8,
            "vote_count": 1
        },
        {
            "adult": false,
            "backdrop_path": null,
            "genre_ids": [],
            "id": 309624,
            "original_language": "en",
            "original_title": "Uncharted Territory: The Story of the 2006-07 Wisconsin Men's Basketball Team",
            "overview": "Join the Wisconsin Badgers' men's basketball team for one of the greatest seasons in their history. This retrospective of the 2006-07 season features highlights from the entire year, as well as interviews with coach Bo Ryan and the team's star players. The dynamic footage from the season that saw the team reach its first-ever No. 1 AP ranking is narrated by \"the Voice of the Badgers,\" Matt Lepay.",
            "popularity": 1.4,
            "poster_path": null,
            "release_date": "",
            "title": "Uncharted Territory: The Story of the 2006-07 Wisconsin Men's Basketball Team",
            "video": false,
            "vote_average": 0,
            "vote_count": 0
        }
    ],
    "total_pages": 1,
    "total_results": 14
}
