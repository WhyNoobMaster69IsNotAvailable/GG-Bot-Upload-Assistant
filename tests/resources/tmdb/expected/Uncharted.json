{
    "tmdb": "335787",
    "imdb": "tt1464335",
    "tvmaze": "0",
    "tvdb": "0",
    "possible_matches": [
        {
            "result_num": 1,
            "title": "Uncharted",
            "content_type": "movie",
            "tmdb_id": 335787,
            "release_date": "2022-02-10",
            "language": "en",
            "overview": "N.A."
        },
        {
            "result_num": 2,
            "title": "Gerry Anderson: A Life Uncharted",
            "content_type": "movie",
            "tmdb_id": 942512,
            "release_date": "2022-04-15",
            "language": "en",
            "overview": "Gerry Andersonâ€™s successful career spanned 60 years. Released a decade after his death, this documentary draws on exclusive access to over 30 hours of previously unpublished interviews, in order to share the untold stories that defined his life and body of work."
        },
        {
            "result_num": 3,
            "title": "Land Uncharted",
            "content_type": "movie",
            "tmdb_id": 962352,
            "release_date": "",
            "language": "en",
            "overview": "N.A."
        },
        {
            "result_num": 4,
            "title": "Uncharted Territory: The Story of the 2006-07 Wisconsin Men's Basketball Team",
            "content_type": "movie",
            "tmdb_id": 309624,
            "release_date": "",
            "language": "en",
            "overview": "Join the Wisconsin Badgers' men's basketball team for one of the greatest seasons in their history. This retrospective of the 2006-07 season features highlights from the entire year, as well as interviews with coach Bo Ryan and the team's star players. The dynamic footage from the season that saw the team reach its first-ever No. 1 AP ranking is narrated by \"the Voice of the Badgers,\" Matt Lepay."
        }
    ]
}